packages/react/src/views/ChatInput/ChatInputFormattingToolbar.js

```diff
@@ -42,7 +43,10 @@ const ChatInputFormattingToolbar = ({
 
   const handleEmojiClick = (emojiEvent) => {
     const [emoji] = emojiEvent.names;
-    messageRef.current.value += ` :${emoji.replace(/[\s-]+/g, '_')}: `;
+    const updatedMessageWithEmoji =
+      messageRef.current.value + ` :${emoji.replace(/[\s-]+/g, '_')}: `;
```

Comment:
i think it's complaining because of this `messageRef.current.value + ` :${emoji.replace(/[\s-]+/g, '_')}: `;`

can we instead do `${messageRef.current.value} :${emoji.replace(/[\s-]+/g, '_')}: `

also, can you check if `messageRef.current.value` can be other than a string? maybe we can add `?? ''` if it's not present

Reply:
fixed it!